components/collapsible-panel/test/collapsible-panel.visual-diff.js

```diff
@@ -47,7 +55,6 @@ describe('d2l-collapsible-panel', () => {
 			{ name: 'default', selector: '#default' },
 			{ name: 'default-focus', selector: '#default', action: focusElement },
 			{ name: 'default-expanded', selector: '#default-expanded' },
-			{ name: 'default-expanded-sticky', selector: '#default-expanded-sticky' },
```

Comment:
The `sticky` tests need `captureBeyondViewport: false`, so just moving them to their own test block